### PR TITLE
Mapbox Sandbox

### PR DESCRIPTION
**Pull request by oliverroick**
Adding a mapbx globe page for demo purposes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,24 +26,27 @@
     "@nextui-org/react": "^2.3.6",
     "@prisma/client": "5.15.0",
     "@statelyai/inspect": "^0.3.1",
+    "@types/mapbox-gl": "^3.4.0",
```
Comment (vgeorge) [inline package.json:29]: @oliverroick can we move this to dev dependencies?

File: src/app/sandbox/mapbox/page.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+"use client";
+import Map, { Layer, Source } from "react-map-gl";
+import "mapbox-gl/dist/mapbox-gl.css";
+
+function MapboxTest() {
+  return (
+    <Map
+      mapboxAccessToken="pk.eyJ1IjoiZGV2c2VlZCIsImEiOiJjbHo5NWh0d3QwMDU0MnBvanAwMmRzYWl0In0.lLN2Ur91TbYhV520kQ3SrA"
```
Comment (vgeorge) [inline src/app/sandbox/mapbox/page.tsx:8]: @oliverroick if we promote this exercise to the real app lets move this token to the .env file.

File: src/app/sandbox/mapbox/page.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      initialViewState={{
+        longitude: 0,
+        latitude: 0,
+        zoom: 2,
+      }}
+      style={{ width: 1000, height: 800 }}
+      mapStyle="mapbox://styles/mapbox/light-v11"
```
Comment (vgeorge) [inline src/app/sandbox/mapbox/page.tsx:15]: @oliverroick can we try not using the basemap from Mapbox and just the MVT boundaries from the local endpoint?

**Review by vgeorge — APPROVED** [review]
@oliverroick thanks for looking into this, I confirm the tiles load fine with mapbox-gl. It seems the MVT layer from Deck.gl is pretty unstable and not ready for using in a production app. I found two related issues in the upstream repository:

- https://github.com/visgl/deck.gl/issues/8870
- https://github.com/visgl/deck.gl/issues/8895

I left some comments to this PR, but this looks like good to merge for demo purposes.

**Comment by oliverroick** [comment]
@vgeorge Addressed the comments. I moved the Mapbox token to a local env and added it to the Amplify env.

**Pull request merged**